Makefile

```diff
@@ -16,6 +16,7 @@ $(warning podman detected but 'podman version' failed. \
 endif
 endif
 
+BUILD_OPTS:=--build-arg BASE_IMAGE=registry.fedoraproject.org/fedora:36
```

Comment:
I would prefer to have different `Containerfile`s and pass it as (optional) argument to `Makefile`. See my comments below.

Reply:
I like the --build-arg approach _iff_ it reduces the delta between supported versions, but otherwise I agree here that it doesn't seem to streamline the approach